.github/workflows/release-appimage.yml

```diff
@@ -0,0 +1,38 @@
+name: Create latest release
+
+on:
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  latest-release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        submodules: recursive
+
+    - name: Build
+      uses: docker://ghcr.io/kdab/kdesrc-build:latest
+
+    - name: Delete latest release
+      run: |
+        gh release delete latest --cleanup-tag || true
```

Comment:
is "latest" the name here? can we call it "continuous" to revive the existing old continuous release from back then?

Reply:
yes that is possible